### DISCUSSION TITLE
How to allow Dropout when predicting

### DISCUSSION THREAD

**Discussion by jpluess97** [MIC-DKFZ/nnUNet#1038] [Q&A]
I am currently looking at the prediction pipeline in order to turn on dropout while testing. I haven't yet found out where this change could be made therefore I wanted to ask for help.

**Comment by FabianIsensee** [comment]
You need to add dropout to the architecture and then just use it in some way. How depends on what you are planning to do, but is should be straightforward to see where in the inference pipeline this should be added (depending on the use-case). Maybe https://github.com/MIC-DKFZ/nnUNet/blob/fd58e25a304e2ab0cd4c16d2c79505c2882e3593/nnunet/network_architecture/neural_network.py#L501 or the 2D equivalent?
